includes/info.h

```diff
@@ -44,17 +44,21 @@ struct InfoGroup {
 };
 
 struct InfoField {
+    const gchar *tag;
     const gchar *name;
     gchar *value;
 
     int update_interval;
 
     gboolean free_value_on_flatten;
+    gboolean mark_for_selection;
 };
 
 struct Info *info_new(void);
+struct Info *info_new_from_str(const gchar *str); /* opposite of info_flatten(); */
```

Comment:
What would be the use case of this one? I intend to kill `info_flatten()` eventually.

Reply:
There's more that's not been pushed yet. info_new_from_str() is needed because I'd started working on a new shell and it uses the info struct exclusively, but so much has not been converted yet. I haven't had any time to work on it this month, I'll try to get back to it.